tests/DataFixtures/ORM/resources/checkout.yml

```diff
@@ -154,3 +157,17 @@ Sylius\Component\Core\Model\Customer:
         email: oliver.queen@star-city.com
         emailCanonical: oliver.queen@star-city.com
         birthday: <date()>
+
+Sylius\Component\Taxation\Model\TaxCategory:
+    regular_tax:
+        code: REGULAR_TAX
+        name: Regular Tax Category
```

Comment:
What would you say for "Taxable Goods"? It's also an example that we use in documentation.